### ISSUE TITLE
Navigation is not working with an a link in an svg figure's g

### ISSUE THREAD

**Issue by MrCsabaToth** [visionmedia/page.js#391]
Example

```
    <div class="panel-body">
        <figure class="imagesvg" style="display: block">
            <svg style="display: block" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" preserveAspectRatio="xMinYMin meet"
                    viewBox="0 0 1212 726" style="max-width: 1212px; max-height: 726px;">
                <image style="display: block" width="1212" height="726" xlink:href="img/clouds.png"></image>
                <g style="display: block" class="hover_area" opacity="0">
                    <a style="display: block" xlink:href="./answers" id="answer_button">
                        <circle cx="592" cy="630" r="84" opacity="0.5" fill="#FF0000"></rect>
                    </a>
                </g>
            </svg>
        </figure>
    </div>
```

Answer is followed, if it is a link in a regular button. But in this case I get an `entry not found` plain text response (and see a 404 in the network tab).
I'm going with the hashbang configuration because I want users to be able to go directly into a sub URL:

```
// Routing
page('/', slideTo);
page('/answers', slideTo);
page('/foo', slideTo);
page('/bar', slideTo);
page({
  hashbang: true
});
```

**Comment by MrCsabaToth** [comment]
Workaround; I was able to attach a click handler to the element and navigate:

    $('#answer_button').click(function(e) {
      page('/answers');
      e.preventDefault();
    });

**Comment by paulocoghi** [comment]
My question is: how can make page.js prevent default clicks automatically in all "a" links, including the ones inside SVG's, without the need to manually attach click handlers to each of them.

**Comment by paulocoghi** [comment]
I identified the issue, and it was inside page.js

I fixed it and I will make a pull request.

**Comment by MrCsabaToth** [comment]
@paulocoghi comment it here when you have the PR. I didn't have time to debug it but I bet it's something to do with the fact that it's an `xlink:href`instead of a `href`

**Comment by paulocoghi** [comment]
@MrCsabaToth Done. https://github.com/visionmedia/page.js/pull/393

If you want, just download from [altiva/page.js](https://github.com/Altiva/page.js) since the project seems to be not maintained.

Also, the problem was more complex than just `xlink:href`. In svg links:

 - the el.nodename property is not `A` but `a`
 - the `href` and `target` are not a string, but an object with the respective value in `el.href.baseVal` and `el.target.baseVal`

**Comment by paulocoghi** [comment]
The point is that with this fix, you don't need to attach a click handler to elements anymore.

**Comment by paulocoghi** [comment]
https://github.com/visionmedia/page.js/pull/394 Updated pull request with a more refined fix.

**Comment by matthewp** [comment]
Thanks for the PR, I'll take a look and try to get it merged soon.

**Comment by matthewp** [comment]
Fixed by #394 (now merged)
